src/SourceBuild/content/Directory.Build.props

```diff
@@ -71,6 +71,8 @@
     <TargetRid Condition="'$(PortableBuild)' == 'true' and '$(PortableRid)' != ''">$(PortableRid)</TargetRid>
     <TargetRid Condition="'$(ShortStack)' == 'true' and '$(TargetOS)' != 'windows'">$(TargetOS)-$(TargetArchitecture)</TargetRid>
     <TargetRid Condition="'$(ShortStack)' == 'true' and '$(TargetOS)' == 'windows'">win-$(TargetArchitecture)</TargetRid>
+
+    <TargetsMusl Condition="$(PortableRid.StartsWith('linux-musl'))">true</TargetsMusl>
```

Comment:
Why not just condition the path changes in runtime.proj on `'$(TargetOS)' == 'linux-musl'`?